ui/src/middlewares/syncLocationWithSearch.js

```diff
@@ -78,7 +81,7 @@ export default function ({ dispatch, getState }) {
         isSearchPage(prevLocation)
       ) {
         dispatch(newSearch(prevNamespace));
-        if (nextNamespace === LITERATURE_NS) {
+        if (ALLOWED_NAMESPACES.includes(nextNamespace)) {
```

Comment:
let's add a more clear name, because it's too generic and we will forget.

Reply:
Done!